.github/workflows/release.yml

```
@@ -0,0 +1,134 @@
+name: Release
+
+on:
+  push:
+    branches:
+        - main
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release'
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: read
+  statuses: write
+  packages: write
+
+jobs:
+  release:
+    name: Release Tuist App
+    runs-on: macos-latest
+    timeout-minutes: 50
+    if: ${{ !contains(github.event.head_commit.message, '[Release]') }}
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+        with:
+          fetch-depth: 0
+      - uses: jdx/mise-action@v2
+        with:
+          experimental: true
+        env: 
+          MISE_HTTP_TIMEOUT: 300
+      - name: Check if there are releasable changes
+        id: is-releasable
+        working-directory: app
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Run git cliff and save the output
+          bumped_output=$(git cliff --include-path "app/**/*" --repository "../" --bump)
+          echo "Bumped output:"
+          echo "${bumped_output}"
+  
+          # Read the content of CHANGELOG.md
+          changelog_content=$(cat CHANGELOG.md)
+          echo "CHANGELOG.md content:"
+          echo "${changelog_content}"
+  
+          # Compare the outputs and set the result
+          if [ "${bumped_output}" = "${changelog_content}" ]; then
+            echo "should-release=false" >> $GITHUB_ENV
+          else
+            echo "should-release=true" >> $GITHUB_ENV
+          fi
+      - name: Get next version
+        working-directory: app
+        id: next-version
+        if: env.should-release == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          NEXT_VERSION=$(git cliff --include-path "app/**/*" --repository "../" --bumped-version)
+
+          if [ "$NEXT_VERSION" = "0.1.0" ]; then
+            NEXT_VERSION="app@0.1.0"
+          fi
+
+          echo "NEXT_VERSION=$NEXT_VERSION" >> "$GITHUB_OUTPUT"
+          echo "NEXT_VERSION_NUMBER=${NEXT_VERSION#*@}" >> "$GITHUB_OUTPUT"
+      - name: Get release notes
+        id: release-notes
+        if: env.should-release == 'true'
+        working-directory: app
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "RELEASE_NOTES<<EOF" >> "$GITHUB_OUTPUT"
+          git cliff --include-path "app/**/*" --repository "../" --unreleased >> "$GITHUB_OUTPUT"
```

Similar comment here. Do we need `--include-path` considering the working-directory is already `app`?
```suggestion
          git cliff --repository "../" --unreleased >> "$GITHUB_OUTPUT"
```

---

This is recommended by `git-cliff`, so I wouldn't do this